.github/workflows/ci.yml

```diff
@@ -0,0 +1,253 @@
+name: Continuous Integration
```

Comment:
This name is a bit too long. Concatenate this name to CI, otherwise it's painful to figure out what workflows are running via mobile.

Reply:
Updated